src/scripts/main.js

```diff
@@ -1,5 +1,60 @@
+/* eslint-disable no-undef */
+// they sad "getComputedStyle is not undefined"
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const spiderWidth = parseFloat(getComputedStyle(spider).width);
+const spiderHeight = parseFloat(getComputedStyle(spider).height);
+
+const wall = document.querySelector('.wall');
+
+spider.style.transitionTimingFunction = 'linear';
+
+const spiderMovings = function(event) {
+  spider.style.left = `${getRightX(event.offsetX)}px`;
+  spider.style.top = `${getRightY(event.offsetY)}px`;
+};
+
+wall.addEventListener('mousemove', spiderMovings);
+
+function getRightX(eventX) {
+  let x = eventX;
```

Comment:
you can pass the variable as 'x', no need to create a separate variable

Reply:
no, eslient dont pass